resources/public/api/conf/2.0/uk_property_non_fhl_retrieve.raml

```diff
@@ -0,0 +1,23 @@
+is:
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+displayName: Retrieve a UK Property Income & Expenses Period Summary
```

Comment:
Right now there are two get request endpoints both with the same name in the same section of the RAML - I wonder if this might be a bit confusing. Might we consider changing the displayName to "Retrieve a Historic Non-FHL UK Property Business Annual Submission" to match the specs? 
<img width="667" alt="image" src="https://user-images.githubusercontent.com/106095575/180187365-5459c081-556e-41e0-8275-d2b60dd25ae2.png">

Reply:
Endpoint name has been resolved